### ISSUE TITLE
Connected glass panes bug

### ISSUE THREAD

**Issue by Jepl4r** [Chocohead/OptiFabric#1011]
Glass panes looks ok only when looking at them facing north or west.

Optifine version is the latest for 1.19.4 and Optifabric is the latest Actions Build version 1.13.24

![2023-03-25_22 53 21](https://user-images.githubusercontent.com/115885800/227744114-198f6882-d376-4d4b-90bc-56300030296b.png)
![2023-03-25_22 53 31](https://user-images.githubusercontent.com/115885800/227744120-44cbd397-bd04-4e8f-922b-c170cbc13b77.png)

**Comment by pooky125** [comment]
Can confirm, I'm also having this issue.

**Comment by Edison946** [comment]
I am also having this issue

**Comment by wiget454** [comment]
Can also see this, on fedora linux 1.19.4
![image](https://user-images.githubusercontent.com/122064539/233470203-b6363b15-1bf7-4492-b6d4-99f5fba61431.png)

**Comment by Jepl4r** [comment]
> Can also see this, on fedora linux 1.19.4
> ![image](https://user-images.githubusercontent.com/122064539/233470203-b6363b15-1bf7-4492-b6d4-99f5fba61431.png)

It’s the same issue, it’s not different

**Comment by RU-B0** [comment]
Just wanted to comment how I fixed this issue: I found out that a mod called fusion (https://modrinth.com/mod/fusion-connected-textures) launches with OptiFine just fine. Using Midnighttigger's Default Connected Textures (https://modrinth.com/resourcepack/mt-ct-d/version/tfKvt8nl) and putting it as lowest priority on the resource pack list (but not below MC itself or fabric), it gets rid of this issue. Plus, OptiFine's implementation still works for other resource packs. Its a win - win. 

Tested with Fusion 1.0.5a and Optifine HD U I5, Optifabric 1.13.25, and FAPI 0.83.1 (+10000000000 other mods but you get the point)
